back/engines/commercial/multi_tenancy/lib/tasks/fixes/ideas_order.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+namespace :fix_existing_tenants do
+  desc 'Fix ideas order after validation constraint changes'
```

Comment:
Is this best as a rake task? Feels like it might be better as a migration given there is no ruby logic involved, so it gets run automatically?

Reply:
I was hesitating, but thought a migration might not be appropriate because we're not doing any schema changes and the release should still succeed. I'll move it to a migration.